dockerfiles/scripts/install_cran_pkgs.R

```diff
@@ -279,6 +277,9 @@ if (length(new_pkgs))
   install.packages(new_pkgs,
                    Ncpus = parallel::detectCores())
 
+# Needs to recompile TMB from source, due to Matrix package version mismatch
+install.packages('TMB', type = 'source')
```

Comment:
I'd move this 2 steps up, before line 269 as `rbmi` should be installed after TMB is installed from source. I'd also add a new vector defining packages that need to be installed from source so we can add additional packages to the vector and add the `Ncpus = parallel::detectCores()` option to the `install.packages` invocation.

Reply:
Sounds good @dinakar29 ! Thanks for the comment